examples/cdip_example.ipynb

```diff
@@ -0,0 +1,629 @@
+      "text/plain": [
+       "<Figure size 720x864 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "buoy_data = cdip.get_netcdf_variables(nc, start_date='2011-01-01', end_date='2011-12-31', parameters='waveHs')\n",
+    "buoy_name = buoy_data[\"data\"][\"wave\"].name\n",
+    "ax = graphics.plot_boxplot(buoy_data[\"data\"][\"wave\"][\"waveHs\"], buoy_title=buoy_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.b. `parse_data`\n",
```

Comment:
This is not a repeat as it uses the `years` parameter instead of the start date/ end date. The `years` parameter is 'advanced' because it simplifies the user inputs and can work on nonconsecutive years if desired. This example also goes one step further by showing the user a plot from the requested data.

I could show off the ability to pass a NetCDF file into it instead if you believe it needs further differentiation.